.fixtures.yml

```diff
@@ -11,5 +11,7 @@ fixtures:
     apt:
       repo: "puppetlabs/apt"
       ref: "2.3.0"
+    yumrepo_core:
+      repo: "puppetlabs/yumrepo_core"
```

Comment:
FYI it's possible to make this conditional for 6.0 and up only, see https://github.com/puppetlabs/puppetlabs_spec_helper/pull/258. I don't think this PR needs to change (since the latest version of yumrepo_core supports older puppet versions), but just something to keep in mind in the future.

Reply:
Thanks! I didn't know that -- I think I'll make a followup PR to do that, just in case.